packages/typespec-ts/README.md

```diff
@@ -153,6 +153,16 @@ If we enable this option `clearOutputFolder` we would empty the whole output fol
 clearOutputFolder: true
 ```
 
+### compatibilityMode
```

Comment:
can you use a new option, in the url template case, compatibilityMode is not fully backward compatible, you still drop the support for ssv and tsv, pipe etc. While in additionaly properties case, it is really backward compatible.

Reply:
updated as `compatibilityQueryMultiFormat` option.